gum/arch-x86/gumx86writer.c

```diff
@@ -832,11 +807,14 @@ gum_x86_writer_put_jcc_short (GumX86Writer * self,
 
 void
 gum_x86_writer_put_jcc_near (GumX86Writer * self,
-                             guint8 opcode,
+                             x86_insn instruction_id,
                              gconstpointer target,
                              GumBranchHint hint)
 {
   gssize distance;
+  guint8 opcode;
```

Comment:
Variables should be declared in the order that they are used, except if one of them represents the return value/result, in which case it goes first.